docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,363 @@
+  <figcaption>An answer from the Data Grid</figcaption>
+</figure>
+
+**Feature's key Benefits:**
+
+- **Enhanced usability**: Simplify interactions with data by asking questions in plain language.
+- **Unlock the full potential of the Data Grid**: Explore, combine, and discover powerful data analysis features often hidden behind menus, without the tedious hopping from one menu to another.
+- **Data privacy:** No need to share your dataset—the service model only requires your column definitions and a few data samples.
+
+Interested in bringing this feature to your users? [Contact us](mailto:sales@mui.com) to get access to your API Key!
+
+### List View [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+We’ve promoted List View to stable, and over this major we’re committed to enhancing this feature even further—making it easier to deliver a robust Data Grid experience on small screens.
+
+<figure>
+  <video autoplay muted loop playsinline width="690" height="417" controls>
+    <source src="/static/blog/mui-x-v8/list-view.mp4" type="video/mp4">
+  </video>
+  <figcaption>Two different views to match the capabilities of multiple displays</figcaption>
+</figure>
+
+Learn more about [List View](/x/react-data-grid/list-view/).
+
+### Server-Side Aggregation and Lazy Loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+The Data Source now supports advanced features such as server-side aggregation, and on-demand data fetching for infinite scrolling, and lazy loading.
+
+Explore all your options when handling [Server-Side Data](/x/react-data-grid/server-side-data/).
+
+### Data Source on the Community Plan
+
+Great news—the Data Source is now part of the community plan!
+
+We're unifying our approach to handling server-side data across all plans, creating a single, cohesive framework.
+
+Explore your options in our [Data Source Docs](/x/react-data-grid/data-source/).
+
+### New Toolbar
+
+The [redesigned toolbar](/x/react-data-grid/components/toolbar/) not only looks fantastic but also empowers developers one step further through an open composition of sub-components, allowing you to tailor the toolbar's layout precisely to your application's needs.
+
+<figure>
+  <img width="100%" alt="The new toolbar's design" src="/static/blog/mui-x-v8/new-toolbar.png" loading="lazy">
+  <figcaption>The new toolbar's design</figcaption>
+</figure>
+
+This new developer experience, based on composition, may feel different from our previous approach—but it sets a new standard. We’ll soon extend this enhanced experience to other key subcomponents, such as the filter panel and columns management panel, ensuring a cohesive and powerful toolkit for advanced customization.
+
+Dive into our new docs section: [Sub-Components Composition](/x/react-data-grid/components/usage/)
```

Comment:
I would make this link reflect the docs page title:

```suggestion
Dive into our new docs section: [Components usage](/x/react-data-grid/components/usage/)
```

Reply:
Don't forget a period! 😁

```suggestion
Dive into our new docs section on [component usage](/x/react-data-grid/components/usage/).
```